### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by bpradipt**
1. Set default nvidia driver version 
2. Update example in AWS README

### CODE REVIEW

File: podvm/addons/nvidia_gpu/setup.sh [hunk 1]
```diff
@@ -39,7 +41,7 @@ if  [[ "$PODVM_DISTRO" == "rhel" ]]; then
     dnf install -q -y kernel-devel-"$(uname -r)" kernel-headers-"$(uname -r)"
 
     dnf install -q -y nvidia-container-toolkit
-    dnf -q -y module install nvidia-driver:latest
+    dnf -q -y module install nvidia-driver:${NVIDIA_DRIVER_VERSION}/default
```
Comment (snir911) [inline podvm/addons/nvidia_gpu/setup.sh:44]: is this valid versioning?

**Review by snir911 — APPROVED** [review]
LGTM, thanks, see comment

File: podvm/addons/nvidia_gpu/setup.sh (hunk 1 above)
Comment (bpradipt) [reply podvm/addons/nvidia_gpu/setup.sh:44]: It's version and stream. `default` is the stream

**Review by mkulke — APPROVED** [review]

**Review by beraldoleal — APPROVED** [review]
LGTM.

**Pull request merged**